pkg/webrtc_ext/webrtc_api.go

```diff
@@ -0,0 +1,57 @@
+package webrtc_ext
+
+import (
+	"fmt"
+
+	"github.com/pion/interceptor"
+	"github.com/pion/webrtc/v3"
+)
+
+// Creates Pion's WebRTC API that has all required extensions configured (such as simulcast).
+func createWebRTCAPI(config Config) (*webrtc.API, error) {
+	mediaEngine := &webrtc.MediaEngine{}
+	if err := mediaEngine.RegisterDefaultCodecs(); err != nil {
+		return nil, fmt.Errorf("failed to register default codecs: %w", err)
+	}
+
+	// Enable extension headers needed for simulcast (if enabled).
+	if config.EnableSimulcast {
```

Comment:
Is the current SFU code actually prepared to work with simulcast disabled?

Reply:
Yep, it should work without the simulcast, albeit with a little overhead of RTP rewriting that would not be necessary (won't harm though). Or at least it worked last time I tried it, I would probably a non-Simulcast EC client to test. We currently have a single place where we check if we need to use layers and it has that `if` block that checks if we have an audio type for a track **or** no layers supplied, then we just use `SimulcastLayerNone` for everyone and the rest works identical.